### PR TITLE
[WPB-7222] Drop depencency on `convertible-strings` in production code

### PR DESCRIPTION
**Pull request by mdimjasevic**
The PR removes all usages of the `cs` function (from `ConvertibleStrings`) in the application code.

Tracked by https://wearezeta.atlassian.net/browse/WPB-7222.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by MangoIV** [comment]
> i would really appreciate if you could drop the habit of reformatting so much to your taste, specifically line add line breaks. if we all keep doing that we'll drown in diff noise.

@fisx this wasn't me this time! :D

File: services/spar/src/Spar/Scim/Auth.hs [hunk 1]
```diff
@@ -38,10 +38,12 @@ where
 import Control.Lens hiding (Strict, (.=))
 import qualified Data.ByteString.Base64 as ES
 import Data.Id (ScimTokenId, UserId)
-import Imports
 -- FUTUREWORK: these imports are not very handy.  split up Spar.Scim into
 -- Spar.Scim.{Core,User,Group} to avoid at least some of the hscim name clashes?
 
+import qualified Data.Text.Encoding as T
```
[the remark this answers is not in the document]
Comment (MangoIV) [reply services/spar/src/Spar/Scim/Auth.hs:41]: alright :)

**Comment by MangoIV** [comment]
Argh. Hoogle upload is stuck again

**Comment by MangoIV** [comment]
@mdimjasevic i think this was more or less directly to me because I have the habit of doing more or less noisy refactoring on the side and we talked about that and I agreed that it’s bad. So I guess that’s why Matthias reminded me to do that less.

**Comment by fisx** [comment]
> I either...

there is a third choice: set up your editor to support line break.  :)

but i get your point.  everything anybody does breaks *somebody*'s workflow...  https://xkcd.com/1172/

(ignore-whitespace on github doesn't work very well.  i suspect it breaks semantics sometimes, but worse, it doesn't catch all the noise that bothers me.)

but i've made my point, and i don't see an easy solution.  feel free to ignore me, then.

**Comment by MangoIV** [comment]
out of all things, ormolu decided to not enforce line length but doesn't adhere to one-line formatting anywhere...

**Comment by mdimjasevic** [comment]
> there is a third choice: set up your editor to support line break. :)

Do you mean line wrapping?


> but i get your point. everything anybody does breaks _somebody_'s workflow... https://xkcd.com/1172/

Fair, it does break my workflow, but I think it's not just mine. We often pair and we screen-share where we always have to increase the font size so the other side can read from their screen, and for these things long lines are just no good.

**Comment by fisx** [comment]
sorry, yes, line wrapping.

i'll shut up now.  :)

**Pull request merged**